.github/workflows/CI-MATs.yml

```
@@ -0,0 +1,114 @@
+  MATS_STATUS:
+    permissions:
+      actions: read
+    needs: [check_build, MATS]
+    runs-on: ubuntu-latest
+    if: ${{ !cancelled() }}
+    outputs:
+      passedMATs: ${{ steps.check_status.outputs.mats_success }}
+      didBUILD: ${{ needs.check_build.outputs.should_run }}
+    steps:
+      - id: check_status
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:31: Double quote to prevent globbing and word splitting

(shellcheck)